client/src/controllers/SlugController.ts

```diff
@@ -10,13 +10,43 @@ import { cleanForSlug } from '../utils/text';
 export class SlugController extends Controller<HTMLInputElement> {
   static values = {
     allowUnicode: { default: false, type: Boolean },
+    applyUpdates: { default: false, type: Boolean },
```

Comment:
```suggestion
```

I do not think we need the slug field to know whether it should be updating as a whole, this should happen per method call.